pkg/ui/src/views/shared/components/dropdown/index.tsx

```diff
@@ -38,19 +38,28 @@ export default class Dropdown extends React.Component<DropdownOwnProps, {}> {
     if (onArrowClick) {
       className += " dropdown--side-arrows";
     }
+
+    let leftClassName = "dropdown__side-arrow";
+    if (_.includes(disabledArrows, ArrowDirection.LEFT)) {
+      leftClassName += " dropdown__side-arrow--disabled";
+    }
+
+    let rightClassName = "dropdown__side-arrow";
+    if (_.includes(disabledArrows, ArrowDirection.RIGHT)) {
+      rightClassName += " dropdown__side-arrow--disabled";
+    }
```

Comment:
I think this would be easier to read with https://www.npmjs.com/package/classnames (which we already vendor)

Reply:
What, we do?  Well you don't say.  I've been meaning to introduce it because I keep being bothered by the same thing in other places.  Since it's not used anywhere that I've seen I just assumed it wasn't involved.

I'll make a follow on PR changing all the random string concatenation to use that.

Reply:
#20314

Reply:
🎉 even better! (It's used in a few places by my `grep` reckoning.)